src/main/java/com/example/glovo/service/OrderService.java

```diff
@@ -0,0 +1,64 @@
+package com.example.glovo.service;
+
+import com.example.glovo.model.Order;
+import com.example.glovo.model.Product;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.*;
+
+@Service
+public class OrderService {
+
+    private Map<Integer, Order> orders = new HashMap<>();
```

Comment:
it is a bad practice to use Integer as key for hashmap because you can lose the data due to collision

Reply:
Thanks, I got it